SEED-platform/seed
# 2690 feat/add annual peak electricity reduction

#### What's this PR do?
Adds AnnualPeakElectricityReduction

#### How should this be manually tested?
Import a BuildingSync property that has AnnualPeakElectricityReduction and see that it's been added under scenarios in on the property detail page, or inspect the database (see screenshots)

#### What are the relevant tickets?
https://github.com/SEED-platform/seed/issues/2690

#### Screenshots (if appropriate)
<img width="1041" alt="Screen Shot 2021-06-03 at 10 31 27 AM" src="https://user-images.githubusercontent.com/564231/120679922-f3b6b100-c456-11eb-90d9-51d617c66883.png">
<img width="1300" alt="Screen Shot 2021-06-02 at 4 36 01 PM" src="https://user-images.githubusercontent.com/564231/120679919-f2858400-c456-11eb-9f42-67247f1d351a.png">
